app/plans/get.controller.js

```diff
@@ -0,0 +1,31 @@
+const displayText = require('./displayText')
+const { getSentencePlanSummary } = require('../../common/data/offenderSentencePlanSummary')
+
+const hasActivePlan = plans => {
+  if (isEmptyObject(plans)) return false
+  let activePlanPresent = false
+  plans.forEach(plan => {
+    if (!plan.completedDate) {
+      activePlanPresent = true
+    }
+  })
+  return activePlanPresent
+}
+
+function isEmptyObject(obj) {
+  return !Object.keys(obj).length
+}
+
+const sentencePlanSummary = async (req, res) => {
```

Comment:
We only ever use id. Worth destructuring out `{ params: id }`

Reply:
agreed